Automattic/wp-calypso
# Fix/enable import no extraneous dependencies for all repo

#### Changes proposed in this Pull Request

Enable eslint rule `import/no-extraneous-dependencies` across the whole repo

#### Testing instructions

* Checkout this branch and run `./node_modules/.bin/eslint-nibble --ext .js,.jsx,.ts,.tsx,.md.js,.md.javascript,.md.jsx .` (it will take a few minutes). Notice how there are no more violations of `import/no-extraneous-dependencies`
